edx_solutions_projects/receivers.py

```diff
@@ -31,3 +33,23 @@ def on_course_deleted(sender, **kwargs):  # pylint: disable=W0613
                 models.WorkgroupUser.objects.filter(workgroup=workgroup).delete()
                 models.Workgroup.objects.filter(id=workgroup.id).delete()
             models.Project.objects.filter(id=project.id).delete()
+
+
+@receiver(post_delete, sender=WorkgroupSubmission)
```

Comment:
Shouldn't this be `sender=WorkgroupUser` to catch an user being deleted?

Reply:
IIRC on the `post_delete` of `WorkgroupUser`, `WorkgroupSubmission` is already deleted. We can try with `pre_delete` though.

Reply:
I tried that but it seems relations are picked up before the `pre_delete` signal is sent and this ends up being deleted anyway. I'll take another look.

Reply:
Thanks!